src/main/java/com/ericsson/ei/queryservice/ProcessFailedNotification.java

```diff
@@ -47,17 +47,17 @@
     private MongoDBHandler handler;
 
     /**
-     * The method is responsible to extract the data on the basis of the
-     * subscriptionName from the Missed Notification Object.
+     * Get a failed notification object by using the subscriptionName key.
      *
      * @param subscriptionName
      * @return List
      */
-    public List<String> processQueryMissedNotification(String subscriptionName) {
+    public List<String> processQueryFailedNotification(String subscriptionName) {
```

Comment:
I think this is very unclear as an interface. If we take only a string then I would believe that it's only one element you can send in. 

Id rather put it as a "post" and put a list in a json body or something. Creating our own format in a string is not suitable in a rest interface in my mind. But ye, this would require changes in other places as well since we do it in other places.